src/Provider/ApiServiceProvider.php

```diff
@@ -42,7 +51,11 @@ public function register()
      */
     protected function setupConfig()
     {
-        $this->mergeConfigFrom(__DIR__.'/../../config/api.php', 'api');
+        $source = realpath(__DIR__.'/../../config/api.php');
+
+        $this->publishes([$source => config_path('api.php')]);
```

Comment:
This looks an awful lot like publishing to me @GrahamCampbell.

It's already done here: https://github.com/dingo/api/blob/develop/src/Provider/LaravelServiceProvider.php#L19